pkgs/servers/ursadb/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchurl, cmake, zeromq, cppzmq }:
 
-stdenv.mkDerivation {
-  name = "ursadb";
+stdenv.mkDerivation rec {
+  pname = "ursadb";
   version = "1.2.0";
 
   src = fetchurl {
```

Comment:
Here the version could also be substituted in the url

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/123438/commits/a8f24f8ed468bf721e5a233401d9cf44c7da6412.